tests/foreman/ui/test_rhcloud_inventory.py

```diff
@@ -666,5 +667,70 @@ def test_rh_cloud_minimal_report(
         # Verify that proper fields are IN report
         required_fields = ['account', 'subscription_manager_id', 'insights_id']
         assert all(all(key in item for key in required_fields) for item in host_data), (
-            "Not all required keys are present in every dictionary"
+            'Not all required keys are present in every dictionary'
         )
+
+
+@pytest.mark.usefixtures('setting_update')
+@pytest.mark.parametrize(
+    'setting_update',
+    ['subscription_connection_enabled=true', 'subscription_connection_enabled=false'],
+    indirect=True,
+)
+def test_verify_generated_report_location(module_target_sat, setting_update):
+    """
+    Verify that the generated report is located in /var/lib/foreman/red_hat_inventory/generated_reports/
+
+    :id: 44b3abb4-c447-4045-abd8-a1c1396a56cb
+
+    :parametrized: yes
+
+    :steps:
+        1. Set the subscription_connection_enabled setting to true/false
```

Comment:
Should we only test when this is set to false, if it's true then the upload job will take over and move it to uploads/done